docs/en/observability/whats-new.asciidoc

```diff
@@ -3,15 +3,109 @@
+Even if cluster users do not have access to the namespace where Elastic Agent is
+deployed, they are able to choose and configure monitoring for a specific pod.
+The following example configuration will enable Elastic Agent to monitor this
+pod using the configuration details for redis without further user action being
+required.
+
+[role="screenshot"]
+image::images/hints-config-example.png[Example annotations configuration for redis]
+
+For more info, https://www.elastic.co/guide/en/fleet/master/elastic-agent-kubernetes-autodiscovery.html#_hints_annotations_based_autodiscover[read the docs].
+
+[discrete]
+== New Istio integration added
+
+With a new Istio integration available in beta for version 8.5, you can collect
+logs from the Istio service mesh with Elastic Agent.
+
+The Istio service mesh extends Kubernetes to establish a programmable
+application-aware network using the powerful Envoy service proxy. Working with
+both Kubernetes and traditional workloads, Istio brings universal traffic
+management, telemetry, and security, to complex deployments. {integration-docs}/istio[Read the docs] for
+more information, and look out for support for metrics coming in future releases.
+
+[role="screenshot"]
+image::images/istio-integration.png[Add the Istio integration to collect logs from the Istio service mesh]
+
+[discrete]
+== View contextual Kubernetes info from APM Service Overview
+
+You can now view Kubernetes-related metadata in the context of a service. From
+the **Service Overview** page, click on the Kubernetes icon to the right of the
+service name to open a popup that contains Kubernetes details such as container
+images, deployments, namespaces, and replicasets.
+
+[role="screenshot"]
+image::images/apm-overview-kubernetes-context.png[View Kubernetes info from APM Service Overview]
+
+You can also view additional information from the **Instance detail** view at the bottom of the page.
+
+[role="screenshot"]
+image::images/apm-services-instance-kubernetes.png[View Kubernetes info from APM Service Overview]
+
+[discrete]
+== APM Storage Explorer available as beta
+
+With Elastic Observability 8.5, APM Storage Explorer is now available as a beta feature.
+The storage explorer enables you to view the total and relative storage footprint
+of APM documents for each instrumented service. This provides visibility into the
+storage impact of configuration changes such as sampling rates, and will help you
+to improve storage requirement forecasts.
+
+[role="screenshot"]
+image::images/apm-storage-explorer.png[View storage footprint with APM Storage Explorer]
+
+[role="screenshot"]
+image::images/apm-storage-explorer-details.png[View storage details for services]
+
+[discrete]
+== Auto-instrument APM .NET Agent profiler
+
+You can now dynamically instrument existing .NET Framework, .NET Core, and .NET
+applications during their startup and gain complete visibility in a matter of minutes.
```

Comment:
```suggestion
You can now dynamically instrument existing .NET Framework, .NET Core, and .NET
applications during startup and gain complete visibility in a matter of minutes.
```

Reply:
I did originally edit that out but then wasn't sure if there was a difference between startup of applications and startup of _something else_ / elastic so was waiting for some PM confirmation. Will take it out again unless I hear something different.